arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/call/CallSection.java

```diff
@@ -123,17 +143,20 @@ public boolean isAbortingScenario() {
   public StpCall stpCall;
   private PrecompileSubsection precompileSubsection;
 
-  @Getter private MemorySpan callProvidedReturnDataTargetSpan;
+  @Getter private MemorySpan returnAtMemorySpan;
+  @Getter private CallDataInfo callDataInfo;
```

Comment:
We snapshot the "call data" and "return at ..." spans in the `CallSection` now. This simplifies the `CallFrame#enter(...)` method down the road.